runtime/litmus/Cargo.toml

```diff
@@ -227,10 +227,6 @@ std = [
   "pallet-teeracle/std",
   "pallet-vc-management/std",
 ]
-tee-dev = [
-  "pallet-teerex/skip-ias-check",
-  "pallet-teerex/skip-scheduled-enclave-check",
-]
```

Comment:
should we also remove 'tee-dev' feature inside runtime/rococo?

Reply:
we can't - node/tee-dev depends on rococo/tee-dev

Reply:
Yes. Just see it above. Is there anything which depends on node/tee-dev? Didn't find any inside parachain source code.

Reply:
node is the binary already so that's the primary build target already